### PR TITLE
USHIFT-1705: feat: re-add /usr/lib check for kustomize

### PR DESCRIPTION
**Pull request by eggfoobar**
added logic to make /usr/lib writable during test, rebooting will revert back to immutable filesystem

local testing didn't show an increase in test time for the kustomize test itself, looking at previous runs, this test was around 2 minutes, and even with the reboot, it seems to hover around there.

### CODE REVIEW

File: test/suites/standard/kustomize.robot [hunk 1]
```diff
@@ -82,17 +84,18 @@ Setup Suite    # robocop: disable=too-long-keyword
...
-    # Set Suite Variable    \${USR_NAMESPACE}    ${ns}
-    #
-    # # Used by "Load From /usr/lib/microshift/manifestsd"
-    # ${rand}=    Generate Random String
-    # Set Suite Variable    \${USR_SUBDIR}    /usr/lib/microshift/manifests.d/${rand}
-    # ${ns}=    Generate Manifests    ${USR_SUBDIR}
-    # Set Suite Variable    \${USR_SUBDIR_NAMESPACE}    ${ns}
+    # and /usr/lib is not writable there, we use `Setup If OSTree Host` to detect
+    # an ostree filesystem and make /usr/lib temporarly writable, we restart during
+    # teardown to go back to initial state.
```
Comment (pmtk) [inline test/suites/standard/kustomize.robot:89]: Do we still need that comment? Would descriptive keyword name for usroverlay and description cover that?

**Comment by pmtk** [comment]
This approach makes sense, beside two small nits, looks good to me

File: test/suites/standard/kustomize.robot [hunk 2]
```diff
@@ -70,6 +71,7 @@ Setup Suite    # robocop: disable=too-long-keyword
     Check Required Env Variables
     Login MicroShift Host
     Setup Kubeconfig    # for readiness checks
+    Setup If OSTree Host    # if ostree make /usr writable for /usr/lib/microshfit/manifest folders
```
[the remark this answers is not in the document]
Comment (eggfoobar) [reply test/suites/standard/kustomize.robot]: That's a good point, changed it to be a more descriptive keyword

File: test/suites/standard/kustomize.robot (hunk 1 above)
Comment (eggfoobar) [reply test/suites/standard/kustomize.robot]: Yeah, maybe not as much needed now, made it more descriptive and removed comment.

**Pull request merged**